src/components/ha-hls-player.ts

```diff
@@ -48,15 +48,19 @@ class HaHLSPlayer extends LitElement {
 
   private _exoPlayer = false;
 
+  private static streamCount = 0;
+
   public connectedCallback() {
     super.connectedCallback();
+    HaHLSPlayer.streamCount += 1;
```

Comment:
I'm not sure how async works in the browser - is it possible that `this._startHLS()` gets deferred, and then there's another call from another element to `connectedCallback` here, so the counter gets incremented twice before `_isLLHLSSupported()` gets called?
Can we just increment this in `_isLLHLSSupported()`?

Reply:
Yes, increment may be called multiple times before the HLS stream is started. When I loaded a lovelace page with 3 streams the counter was 3 before the first call to load the stream. I actually thought it was a nice side effect that it happened to work that way, but open to changing.

I am definitely open to moving this somewhere else, though _isLLHLSSupported seems like it would be non-obvious that it has that side effect so easy to break in the future. Also it seems hard to know where to decrement. I chose this spot because it is easy to see the symmetry for connect and disconnect and i'd have reasonable confidence it works consistently.

Reply:
I guess in this case you're right the race is in our favor, because 2 LL-HLS + 1 HLS might exceed capacity. Let's add a comment to show the intention.

Reply:
I don't think it's that strange to have it incremented where it is actually checked. Although I agree with your point that it is cleaner to do it symmetrically.
Although you're right - I was thinking of the case where someone has say 4 videos and thinking that 2 would be allowed to be LL-HLS - on second thought I think that is pushing the envelope too far anyway.
To circle back though I will note the case where someone has say 8 picture entity cards (with large still images that refresh every 10 secs or so). This user may still have issues as when all those images go to reload, they will use up the connection pool temporarily. Given this case, is it still worth it to allow for this 2 stream exception?

Reply:
I think that's fine. Temporarily broken should be recoverable / be covered by a buffer?

Reply:
I think hls.js should be able to handle it and should adjust the live edge accordingly, but no guarantees